### DISCUSSION TITLE
Is there a way to make the button focused when active?

### DISCUSSION THREAD

**Discussion by maralihart** [microsoft/fluentui#27251] [Q&A] [answered]
Like how with the Switch, checked determines whether it's checked or not

**Comment by behowell** [comment]
Hi @maralihart is the [ToggleButton](https://react.fluentui.dev/?path=/docs/components-button-togglebutton--default) component what you're looking for? If not, could you give some more details about what you're asking for; possibly example code with a description of how you would expect it to work?

**Comment by maralihart** [comment]
```
const [active, setActive] = useState(true)

return (<Button active={active}>Content stays focused even when user stops
clicking</Button>)
```

Expected view:
Screen Shot 2023-03-27 at 9.25.43 AM.png
<https://mail.google.com/mail/u/0?ui=2&ik=ce3ac29e50&attid=0.1&permmsgid=msg-a:r-2219850335160813928&view=att&disp=safe&realattid=f_lfr1jxdy0>
(8K)
<https://mail.google.com/mail/u/0?ui=2&ik=ce3ac29e50&attid=0.1&permmsgid=msg-a:r-2219850335160813928&view=att&disp=safe&realattid=f_lfr1jxdy0>

**Comment by behowell** [comment, marked as answer]
I'm not able to access the screenshot you linked. But based on the code you pasted, [ToggleButton](https://react.fluentui.dev/?path=/docs/components-button-togglebutton--default) sounds like what you want. The prop for active is called `checked`.
